astropy/io/fits/tests/test_table.py

```diff
@@ -976,7 +975,7 @@ def test_add_data_by_rows(self):
         assert tbhdu.columns.columns[2].array[0] == ''
         assert (tbhdu.columns.columns[3].array[0] ==
                 np.array([0., 0., 0., 0., 0.], dtype=np.float32)).all()
-        assert tbhdu.columns.columns[4].array[0] == True
+        assert tbhdu.columns.columns[4].array[0] == True  # nopep8
```

Comment:
Ahh, I shouldn't have done it there either as per a previous comment (https://github.com/astropy/astropy/pull/4581/commits/df55f50e3c999085f589653ec24e868807b5d3d6#r52209960) and since the bool value is needed to be tested, too.